pswi/02_ptf.sh

```diff
@@ -33,7 +33,7 @@ ADD_WORKFLOW_JSON='{"workflowName":"'$PTF_WF_NAME'",
 {"name":"DISTRIBUTION","value":"'$DZONE'"},
 {"name":"PTF1","value":"'$PTF1'"},
 {"name":"PTF2","value":"'$PTF2'"}
```

Comment:
I expect 1 or 2, for now, that's why there is the PTFNR variable. I am not really experienced with PTFs so that's how I handled it. If there would be just one PTF, the PTF2 variable will be just empty.